api/test/data/test_redflags.py

```diff
@@ -92,6 +135,14 @@ def activate_account_and_login(self, activate_data=''):
 
         return self.response.data['token']
 
+    def delete_redflag(self, url='/api/redflags/int:redflag-id/'):
```

Comment:
You should not specify the type(`int`) to the red-flag -d

Reply:
I have implemented this requested change